modules/browser/BrowserCapabilities.js

```diff
@@ -270,7 +279,15 @@ export default class BrowserCapabilities extends BrowserDetection {
      * @returns {boolean}
      */
     usesAdapter() {
-        return this.usesNewGumFlow();
+        return !this.isFirefox() && !this.isReactNative();
```

Comment:
why not on Firefox?

Reply:
Ok , will revert this for now. I will bump up the adapter to 7.5.0 after I test it on all browsers.